docs/resources/project_issue.md

```diff
@@ -55,7 +55,6 @@ output "welcome_issue_web_url" {
 
 - `assignee_ids` (Set of Number) The IDs of the users to assign the issue to.
 - `closed_at` (String) When the issue was closed. Date time string, ISO 8601 formatted, for example 2016-03-11T03:45:40Z.
-- `closed_by_user_id` (Number) The ID of the user that closed the issue. Use `gitlab_user` data source to get more information about the user.
 - `confidential` (Boolean) Set an issue to be confidential.
 - `created_at` (String) When the issue was created. Date time string, ISO 8601 formatted, for example 2016-03-11T03:45:40Z. Requires administrator or project/group owner rights.
```

Comment:
Did you miss some? I don't see how `created_at`, `closed_at`, or `updated_at ` can also be Optional, for example.

Reply:
Indeed, I've missed `closed_at`. However, both [`created_at`](https://docs.gitlab.com/ee/api/issues.html#:~:text=is%20false.-,created_at,-string) and [`updated_at`](https://docs.gitlab.com/ee/api/issues.html#:~:text=of%20an%20issue-,updated_at,-string) can be set using the created and update API endpoints, respectively ..

Reply:
How bizarre 😆